error TS2349: Cannot invoke an expression whose type lacks a call signature

Testing with angular, ionic, and typescript.

Test Repo: https://github.com/peterennis/aemojitest
It works using the script load from unpkg

I want to lazyload.

Failure on branch "angulartest" here: https://github.com/peterennis/aemojitest/tree/angulartest

```
C:\ae\aemojitest>npm start

> aemojitest@0.0.1 start C:\ae\aemojitest
> ng serve

 10% building 3/3 modules 0 activei ｢wds｣: Project is running at http://localhost:4200/webpack-dev-server/
i ｢wds｣: webpack output is served from /
i ｢wds｣: 404s will fallback to //index.html

chunk {main} main.js, main.js.map (main) 1.97 kB [initial] [rendered]
chunk {polyfills} polyfills.js, polyfills.js.map (polyfills) 149 kB [initial] [rendered]
chunk {runtime} runtime.js, runtime.js.map (runtime) 6.09 kB [entry] [rendered]
chunk {styles} styles.js, styles.js.map (styles) 116 kB [initial] [rendered]
chunk {vendor} vendor.js, vendor.js.map (vendor) 339 kB [initial] [rendered]
Date: 2019-09-27T00:39:13.437Z - Hash: 61cab9594fdca97e6e1e - Time: 8379ms

ERROR in node_modules/emojisplosion/src/utils.ts:8:11 - error TS2349: Cannot invoke an expression whose type lacks a call signature. Type '(() => T) | (T & Function)' has no compatible call signatures.

8         ? value()
            ~~~~~~~

** Angular Live Development Server is listening on localhost:4200, open your browser on http://localhost:4200/ **
i ｢wdm｣: Failed to compile.
```

![Capture680](https://user-images.githubusercontent.com/140737/65734037-feb41e00-e085-11e9-8546-a009d338d0d4.png)

--- a/.npmignore
+++ b/.npmignore
@@ -1,4 +1,6 @@
-node_modules/
-test/
 *.test.*
+*.ts
+!*.d.ts
+node_modules/
 npm-debug.log
+test/

